config/default/core.entity_view_display.node.localgov_event.teaser.yml

```diff
@@ -36,6 +37,11 @@ content:
     third_party_settings: {  }
     weight: 2
     region: content
+  content_moderation_control:
```

Comment:
Content moderation control should be hidden on all display modes.

Reply:
This should be fixed now.